### PR TITLE
Update README

### LINKED ISSUE #3: Update README
Add the following sections to README:
- Introduction, including the WHY of the project
- Contributing guideline
- Contact information
- License information

### PR DESCRIPTION
**Pull request by lapanti**
Fixes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,63 @@
 # A very opinionated frontend boilerplate
 [![Build Status](https://travis-ci.org/Lapanti/ts-react-boilerplate.svg?branch=master)](https://travis-ci.org/Lapanti/ts-react-boilerplate) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) [![Dependency Status](https://gemnasium.com/badges/github.com/Lapanti/ts-react-boilerplate.svg)](https://gemnasium.com/github.com/Lapanti/ts-react-boilerplate)
 
-## Dependencies
+## Purpose
+
+This is all you need to get started in developing your own web application, using TypeScript, React, server-side rendering and all the other hip tools. All the source code is well-documented and explains it's purpose. You can read the [getting started guide here](#getstarted)
```
Comment (olliahonen) [inline README.md]: Should say "its", not "it's"

File: README.md [hunk 2]
```diff
@@ -1,7 +1,63 @@
...
+2. Run `yarn` in the console to install dependencies (it'll take a while on the first run, so go on and read ahead while you wait)
+3. Read through the comments in all the source files to get yourself acquinted with the ideas, concepts and patterns
+4. Start the application by running `yarn run develop` in your console (inside the folder you downloaded the code to) and open up your browser in the address it prints out
+    - If you want the use of [LiveReload](http://livereload.com/), install a browser extension in [Chrome](https://chrome.google.com/webstore/detail/livereload/jnihajbhpnppcggbcgedagnkighmdlei?hl=en) or [Firefox](https://addons.mozilla.org/en-gb/firefox/addon/livereload/)
+5. Create a deployable version of the application by running `yarn run build`
+6. Start the deployable version by running `yarn run start` or read the [How to Docker](#dockerization) guide to Dockerize your application
+7. To test your application, run `yarn run test`
+8. Start modifying the code to build your own application
+9. ???
```
Comment (olliahonen) [inline README.md]: I would leave `9.` and `10.` out :) Just to keep it short

File: README.md [hunk 3]
```diff
@@ -1,7 +1,63 @@
...
+
+## <a name="dockerization">How to Docker</a>
+The [Dockerfile](Dockerfile) is where you can find the configuration to build a [Docker](https://www.docker.com/) image out of your application. The first line of the `Dockerfile` (starting with `FROM`) includes the base for your Dockerfile, feel free to change it if you want to.
+1. Put your email to the [fourth line in the Dockerfile](Dockerfile#L4)
+2. In your console run `docker build .`
+3. In your console run `docker run -d -p 8888:8888 bd9b1d6725bc` **but** replace `bd9b1d6725bc` with the image ID you received from the previous command
+4. Host your Docker image in your favorite cloud or local server (the web is filled with guides for this)
+
+## <a name="dependencies">Dependencies</a>
+The following are all the dependencies of the project, with the reasoning behind their includement:
```
Comment (olliahonen) [inline README.md]: `s/includement/inclusion/` or `s/ing behind their includement/ for including them/`

File: README.md [hunk 4]
```diff
@@ -1,7 +1,63 @@
...
+## <a name="tipsandsuggestions">Tips and suggestions</a>
+- Make sure everything has a type (the more you squeeze out of the compiler the easier you're going to have it while developing)
+- Follow [BEM](http://getbem.com/naming/)-naming with CSS
+- Follow [Redux-ducks pattern](https://github.com/erikras/ducks-modular-redux) except that name the reducers as according to the file (see [IndexReducer.tsx](src/modules/index/IndexReducer.tsx) for an example)
+
+## <a name="dockerization">How to Docker</a>
+The [Dockerfile](Dockerfile) is where you can find the configuration to build a [Docker](https://www.docker.com/) image out of your application. The first line of the `Dockerfile` (starting with `FROM`) includes the base for your Dockerfile, feel free to change it if you want to.
+1. Put your email to the [fourth line in the Dockerfile](Dockerfile#L4)
+2. In your console run `docker build .`
+3. In your console run `docker run -d -p 8888:8888 bd9b1d6725bc` **but** replace `bd9b1d6725bc` with the image ID you received from the previous command
```
Comment (olliahonen) [inline README.md]: This one's actually my bad: both of the `8888` should be `8080`

File: README.md (hunk 3 above)
Comment (lapanti) [reply README.md]: @olliahonen fixed (all of the above)

**Comment by lapanti** [comment]
@olliahonen your suggested fixes have been implemented

**Review by olliahonen — APPROVED** [review]

**Pull request merged**